etl_extension/org_unit_etl_extension.sql

```diff
@@ -12,7 +12,16 @@ BEGIN
     ALTER TABLE patient
```

Comment:
Remove "extension" from file name

Reply:
They are named as procedures, which they represent. Instead of renaming I've moved them into a separate folder.

Reply:
Ok.